Default (Linux).sublime-keymap

```diff
@@ -124,6 +124,14 @@ LaTeX Package keymap for Linux
 			{"key": "lt_fill_all_env.open_curly", "operator": "equal", "operand": true, "match_all": true}],
 		"command": "latex_fill_all", "args": {"completion_type": "env", "insert_char": "{"}},
 
+	{	"keys": ["="],
+		"context":  [
+			{"key": "setting.disable_latex_ref_cite_auto_trigger", "operator": "not_equal", "operand": true},
+			{"key": "selector", "operator": "equal", "operand": "text.tex.latex comment.line.percentage.tex"},
```

Comment:
On ST2, the scope for these directives is `comment.line.percentage.directive.texshop.tex`, which means that the autocompletions do not work. It should probably be changed to either `comment.line.percentage` or just `comment`.